### ISSUE TITLE
Include a pre-trained model example?

### ISSUE THREAD

**Issue by tmcw** [developmentseed/label-maker#65]
Hi folks!

Over at @observablehq we've been having fun using TensorFlow.js to run lightweight ML models in-browser, like [in this recent example](https://beta.observablehq.com/@nsthorat/visualizing-activations-of-mobilenet-with-tensorflow-js).

I'd love to tinker with the same idea with a map-trained model, and wonder if the project might include a TensorFlow SavedModel or Keras pre-trained model as a starting point, or as part of the detailed walk-throughs. Would this be in-scope of the project, as a way to play around with the model's activations without needing to setup & train it from scratch?

**Comment by drewbo** [comment]
Hi @tmcw!

Huge fan of @observablehq. I'd say the pre-trained models are "scope-adjacent" in that we definitely want to host examples somewhere but maybe not in this repo specifically. We've also run a few TensorFlow.js experiments locally and we should be able to open up some code around that soon

**Comment by tmcw** [comment]
👍 Thanks Drew (sorry for disappearing!). I assume the folks at ꙩ will update if they need anything more.
